src/main/java/net/dv8tion/jda/core/entities/impl/EmoteImpl.java

```diff
@@ -220,4 +221,13 @@ public String toString()
     {
         return "E:" + getName() + '(' + getIdLong() + ')';
     }
+
+    public EmoteImpl clone()
+    {
+        if (isFake()) return null;
+        EmoteImpl copy = new EmoteImpl(id, guild).setManaged(managed).setName(name);
+        copy.roles.addAll(roles);
```

Comment:
If the `guild` is null this is a fake emote, which means we don't copy it `if (isFake()) return null;`